node/core/dispute-coordinator/src/real/initialized.rs

```diff
@@ -66,6 +71,9 @@ use super::{
 	OverlayedBackend,
 };
 
+const LRU_SCRAPED_BLOCKS_CAPACITY: usize = 100;
```

Comment:
could you add a comment on how it was chosen? we currently allow unfinalized deepth of 500